Content.Server/GameObjects/Components/Items/FloorTileItemComponent.cs

```diff
@@ -21,12 +22,15 @@ public class FloorTileItemComponent : Component, IAfterInteract
         [Dependency] private readonly IMapManager _mapManager = default!;
```

Comment:
I know it's not part of the PR but do you mind resolving this when needed rather than as a member variable? Given there's a lot of loose tiles on the station it'd be decent memory savings.

Reply:
No idea how to do that